scripts/zap-scanner.sh

```diff
@@ -1,12 +1,13 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
There are subtle nuances between these shells, and I ran into some issues using arrays with `sh`.